docs/release-notes/16-6-0/README.md

```diff
@@ -3,24 +3,103 @@ title: OpenProject 16.6.0
+
+![OpenProject 16.6: Project overview page showing three project attributes: Benefit = 10, Effort = 60, Overall Score (calculated) = automatically set to -1,2](openproject-16-6-project-attributes-overview-page.png)
+
+### Performance updates
+
+OpenProject 16.6 introduces several backend optimizations that significantly improve performance in large environments. [Database queries for collection endpoints in the API v3 have been optimized](https://community.openproject.org/wp/68457) to avoid unnecessary counting operations, and the [autocompleter for adding work package relations now requests only the data it actually needs](https://community.openproject.org/wp/68458).
+
+These improvements reduce query load and shorten response times, especially for installations with thousands of projects and work packages.
+
+### New index page for Documents module
+
+With OpenProject 16.6, a new index page provides a structured overview of all documents within a project. The list is sorted by last edited, showing the latest changes first, and includes columns for Name, Type, and Last edited. Users can search documents via a quick text filter or narrow results by document type using the filter menu on the left. 
+
+A new Create document button lets users quickly add new items, with Note set as the default type. Of course, the view automatically respects project permissions, ensuring that users only see documents they are allowed to access. On mobile, the list is optimized to show just the most relevant information.
+
+While this is a small update by itself, it marks the beginning of a major improvement of the Documents module, which will make managing and [live collaborating on documents in OpenProject](https://community.openproject.org/wp/66309) much easier in the future.
+
+![OpenProject 16.6: Redesigned index page for the Documents module, showing a table with clickable name, type and last edited.](openproject-16-6-documents-index-page.png)
+
+### Possibility to change parent of a custom field item
+
+Administrators can now change the parent of an item within a hierarchical custom field. This makes it easier to rearrange existing items without recreating them from scratch. 
+
+To do so, administrators need to navigate to *Administration → Custom fields*, select a custom field type hierarchy and click on the *Items* tab. Then they click on the *More* icon and select *Change parent*. A dialog opens showing the current hierarchy tree. From there, administrators can search, select a new parent, and save the updated structure. The hierarchy is updated immediately after saving.
+
+![OpenProject 16.6: Custom field type hierarchy in the administration, tab 'Items', one item is selected with the 'More' menu and the option to change parent is higlighted](openproject-16-6-custom-field-hierarchy-change-parent.png)
+
+### Updated 'More' menu of meetings with a 'Add to section' option
```

Comment:
> Updated 'More' menu of meetings with a 'Add to section' option

should be

> Updated 'More' menu of meetings with an 'Add to section' option

with an 'an'  instead of the 'a' given that what follows starts with a vowel.